src/components/Modal/SendToken.vue

```diff
@@ -70,6 +80,46 @@ function handleMaxClick() {
   }
 }
 
+function handleSubmit() {
+  const baseAmount = parseUnits(
+    form.amount.toString(),
+    currentToken?.value.contract_decimals
+  );
+
+  const isEth =
+    currentToken?.value.contract_address ===
+    '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee';
+
+  let data = '0x';
+  if (!isEth) {
+    const iface = new Interface(erc20);
+    data = iface.encodeFunctionData(
+      'safeTransferFrom(address, address, uint256)',
```

Comment:
This should use `transfer` method, `safeTransferFrom` is for NFTs
```suggestion
      'transfer(address, uint256)',
```